features/step_definitions/user_steps.rb

```diff
@@ -212,24 +202,38 @@ def log_in_as(username)
 end
 
 Given('they do not have a phone') do
-  choose('select_phone_form_mobile_phone_false', allow_label_click: true)
-  click_on('Continue')
+  uncheck "A phone or tablet that can download an app", allow_label_click: true
+end
+
+Given('they do have a phone') do
+  check "A phone or tablet that can download an app", allow_label_click: true
 end
 
 Given('they continue to register with IDP {string}') do |idp|
   click_on("Choose #{idp}")
+  @idp = "#{idp}"
+end
+
+Given('they continue to register with LOA2 IDP {string}') do |idp|
```

Comment:
This new step is functionally equivalent to the step above. Is it just to be clear about what's happening? I found it a little confusing.